### PR TITLE
Hotfix api endpoints

### PR DESCRIPTION
**Pull request by martinjonsson01**
## Summary
Defines and implements a web API endpoint naming convention.

### CODE REVIEW

File: src/Server/Controllers/ContractsController.cs [hunk 1]
```diff
@@ -87,7 +87,7 @@ public IActionResult CreateContract(Contract contract)
     /// </summary>
     /// <param name="id">Id of the contract to be removed.</param>
     /// <returns>If the contract was successfully removed.</returns>
-    [HttpDelete("{id:guid}")]
+    [HttpDelete("{id:Guid}")]
```
Comment (JacobBredin) [inline src/Server/Controllers/ContractsController.cs:90]: Microsoft uses a lower case guid, but i guess it doesnt matter:

https://docs.microsoft.com/en-us/aspnet/core/fundamentals/routing?view=aspnetcore-6.0#route-template-reference

**Review by JacobBredin — APPROVED** [review]
Looks good, just got one question

File: src/Server/Controllers/ContractsController.cs (hunk 1 above)
Comment (martinjonsson01) [reply src/Server/Controllers/ContractsController.cs]: You're right, I misremembered!

**Review by Matte02 — APPROVED** [review]
LGTM!

**Review by XwayTomy — APPROVED** [review]
Looks resonable! GJ!

File: src/Infrastructure/Contracts/FakeContractRepository.cs [hunk 2]
```diff
@@ -38,7 +38,7 @@ public void Add(Contract contract)
     /// <inheritdoc />
     public void AddRecent(Contract contract)
     {
-        _recent.AddRecent(contract);
+        _recent.Add(contract);
```
Comment (jonathan-lindqvist) [inline src/Infrastructure/Contracts/FakeContractRepository.cs:41]: 👍

**Review by jonathan-lindqvist — APPROVED** [review]
LGTM!

**Pull request merged**